packages/core/src/outcomes.ts

```diff
@@ -181,7 +181,7 @@ export function badRequest(details: string, expression?: string): OperationOutco
         details: {
           text: details,
         },
-        expression: expression ? [expression] : undefined,
+        ...(expression ? { expression: [expression] } : undefined),
```

Comment:
🤔 

I suppose this avoids setting `expression: undefined`, but does that change anything in practice?

Reply:
Yes, when I tried to compare a `badRequest` from over the wire with a `badRequest` instantiated, the strict equals failed since the newly created one had the extra `expression` key

Reply:
Since we remove keys with `undefined` values when sending over the wire